hummingbot/connector/exchange/kucoin/kucoin_api_order_book_data_source.py

```diff
@@ -345,7 +347,7 @@ async def get_new_order_book(self, trading_pair: str) -> OrderBook:
 
     async def get_markets_per_ws_connection(self) -> List[str]:
         # Fetch the  markets and split per connection
-        all_symbols: List[str] = await self.fetch_trading_pairs()
+        all_symbols: List[str] = self._trading_pairs if self._trading_pairs else await self.fetch_trading_pairs()
```

Comment:
In place of this check, replace `self.fetch_trading_pairs()` with `self.get_trading_pairs()` which does the same check as this if and keeps the checks in the correct place?

Reply:
So as you don't need the `self._trading_pairs if self._trading_pairs else`